src/openeo_grass_gis_driver/jobs_job_id.py

```diff
@@ -80,10 +80,32 @@ def get(self, job_id):
 
                         # Store the updated job in the database
                         self.job_db[job_id] = job
-                else:
+                elif code == 400:
+                    # actinia response contains only status and message
                     if job.additional_info != job_info:
                         job.additional_info = job_info
+                        if job_info["status"] == "finished":
+                            job.status = "finished"
+                        if job_info["status"] == "error":
+                            job.status = "error"
+                        if job_info["status"] == "accepted":
+                            job.status = "queued"
+                        if job_info["status"] == "terminated":
+                            job.status = "canceled"
+                        if job_info["status"] == "running":
+                            job.status = "running"
+
+                        # Store the updated job in the database
                         self.job_db[job_id] = job
+                else:
+                    # other 4xx errors
+                    if job.additional_info != job_info:
+                        job.additional_info = job_info
+
+                    job.status = "error"
```

Comment:
This might be dangerous. IIUC, the job status is always updated here on http codes not `200` or `400`. This way, the job could be set to `error` if actinia is temporarily not available and later on a new request be set to `running` again. Or even to `error` if actinia returns other `2xx` succces codes (which I wouldn't know about in this context). I would vote for an `unclear` option if it would be allowed in openeo api :D
But maybe `error` is the best option here, although above objections.